modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Qt5CPPGenerator.java

```diff
@@ -266,6 +267,8 @@ public String toDefaultValue(Property p) {
             return "0.0";
         } else if (p instanceof FloatProperty) {
             return "0.0f";
+        } else if (p instanceof BaseIntegerProperty) {
```

Comment:
@YetAnotherMinion thanks for the PR. I would put this line after the IntegerProperty and LongProperty `else if` check as both IntegerProperty and LongProperty extends BaseIntegerProperty.